src/storage/Cargo.toml

```diff
@@ -67,6 +67,7 @@ tokio = { version = "0.2", package = "madsim-tokio", features = [
     "signal",
 ] }
 tokio-retry = "0.3"
+tonic = { workspace = true }
```

Comment:
We will use tonic(`tonic::transport::Channel;`) in storage folder.

Reply:
We tend to put all rpc client implementation in src/rpc_client. Can we put GrpcProxyClient there insated? To avoid confusion, we should call it GrpcCompactorProxyClient.

Reply:
Previously storage crate only maintain all rpc client, but this pr introduced a rpc server, so we need to add tonic in storage crate.